src/components/dropdown/Dropdown.tsx

```diff
@@ -162,32 +169,20 @@ export class Dropdown<T = LabelValue> extends React.Component<
       onMenuOpen,
       onMenuClose,
       optionRenderer,
-      tagRenderer,
-      tooltip,
-      tooltipCloseLabel,
       showRequired,
+      size,
       tabSelectsValue,
+      tagRenderer,
       termSearchMessage,
+      tooltip,
+      tooltipCloseLabel,
       value,
-      defaultOptions,
-      menuPlacement,
-      menuShouldScrollIntoView,
-      menuPortalStyles,
-      menuPortalTarget,
-      menuShouldBlockScroll,
+      variant,
       ...otherProps
     } = this.props;
 
     return (
-      <div className={className}>
-        <LabelTooltipDecorator
-          htmlFor={id}
-          label={label}
-          placement={'top'}
-          tooltip={tooltip}
-          tooltipCloseLabel={tooltipCloseLabel}
-          showRequired={showRequired}
-        />
+      <div className={classNames(className, `${prefix}-group`)}>
```

Comment:
What's the group?

Reply:
le group is the wrapper of the dropdown and the label